### DISCUSSION TITLE
Lost wallet backups in the process of updating from defi-app 2.3.0 to DeFi Wallet 2.3.2

### DISCUSSION THREAD

**Discussion by UglyJo** [DeFiCh/app#717] [Q&A]
Hello everybody,

During the transition from v2.3.0 to v2.3.2 on Win 10, I lost my wallet, my backups and probably all my coins.
Before the transition, I copied the wallet folder with all my wallet backups from DEFI Blockchain folder under defi-app onto my OneDrive, at least that was what I though I did. Then I installed the DeFi Wallet and tied to load my previously saved wallets. And I deinstalled defi-app 2.3.0.

It turned out every one of these wallets in the wallet folder were only the automatic created wallets, and when I tried to open them one by one they were all empty! So where were the wallet-backups?! I reached out to the Telegram DeFiChain community, and JJ and Bella tried to help me to recover the wallet or the key. But we were unable to do so, and we did not find them.
Thanks so much JJ and Bella!

So, over night I pulled all the previously deleted files out of the Recycle Bin. And I thought, it would be a great idea to try to re-install the defi-app 2.3.0 in order to recreate the key or to retrace where the backup wallets would have gone. And with that I overwrote the wallet backs in C:\Users\Joachims_Dell_XPS15\AppData\Local\Programs\defi-app\resources\binary\win\wallet\wallet_backup_20210327.dat ! 🤦‍♂️

I am running right now the 3rd or 4th recovery software, and I found the links to the wallet_backups, but the actual wallet_backups are gone.

Any idea what I can do now? You guys are my last hope!

Cheers,

Jo

**Comment by thomas425777** [comment]
> and when I tried to open them one by one they were all empty!

How did you check they were empty? (I think the balance appears only when the app is synchronized with the block chain up to the point where your transactions are. Synchronizing from zero can take a long time if you do not use a snapshot)

Regards

**Comment by UglyJo** [comment]
Yes, the new DeFi Wallet 2.3.2 synchronized completely with the blockchain after installation. As you said it took a while, and then I loaded the wallets one by one. But they were empty because they were only the automatically created wallets which were saved when the old defi app 2.3.0 could not sync completely with the blockchain. 

Cheers,

Joachim
